### ISSUE TITLE
Absence Attribute Vide

### ISSUE THREAD

**Issue by samszo** [IEMLdev/ieml#53]
Dans https://intlekt.io/api/dictionary/M:.I:.n.-/relations/
Pourquoi l'attribut E: n'est pas mentionnée dans etymology.father.Attribute alors qu'il compose bien la table ?

**Comment by ogrergo** [comment]
Effectivement, il devrait être là : mon calcul des relations étymologique ignore `E:`, ce qui n'est pas correcte ici.
J'avais fait ça parce qu'il y a aussi d'autres `E:` qui ne doivent pas être mentionné dans l’étymologie:
Le script `M:.I:.n.-` est la version abrégée de `M:E:E:.I:E:E:.n.-`
Dans ce paradigme on trouve par exemple `M:.E:.n.-` en première colonne, qui est la version abrégée de `M:E:E:.E:E:E:.n.-`
La substance est `M:E:E:.` et les `E:` en attribut et en mode ne compte pas dans l’étymologie.
L'attribut est `E:E:E:.` et sa substance est `E:` qui doit être comptée dans l’étymologie.
Donc c'est bien un bug

**Comment by ogrergo** [comment]
La règle pourrait être: on ignore les `E:` dans l'étymologie d'un script quand ils sont présents à la même position dans le paradigme racine de ce script

`M:E:E:.I:E:E:.n.-` paradigme racine
`M:E:E:.E:E:E:.n.-` 1ere colonne
`M:_:_:.E:_:_:.n.-` étymologie

**Comment by plevyieml** [comment]
Bravo Louis et Samuel!
